src/cases/components/constants/index.ts

```diff
@@ -0,0 +1,5 @@
+import { ROLE_ALLOCATION_CONSTANTS as RoleAllocationConstants } from './allocation.constants';
+
+export {
+  RoleAllocationConstants
+};
```

Comment:
would this best to move to src\work-allocation-2\components, if this is only used in work allocation 2.